website/content/en/docs/building-operators/golang/quickstart.md

```diff
@@ -34,6 +34,12 @@ Create a simple Memcached API:
 operator-sdk create api --group cache --version v1 --kind Memcached --resource=true --controller=true
 ```
 
+**Note** If your cluster is a non-vanilla one then such as OpenShift then, add the RBAC finalizer permission on the `controllers/memcached_controller.go` file and run `make manifests` before continue. 
+
+```go
+// +kubebuilder:rbac:groups=cache.example.com,resources=memcacheds/finalizers,verbs=update
+```
```

Comment:
A couple of notes:
1. We shouldn't put such specific information in the quickstart. Something like this seems more relevant to an FAQ entry (e.g. "I saw error X in my logs. How do I fix it?")
1. The need for this permission is based on the [OwnerReferencesPermissionEnforcement](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#ownerreferencespermissionenforcement) plugin, which can be enabled in any Kubernetes cluster because it is a feature of the upstream kube-apiserver. Anytime we talk about the need for this permission, we should reference that plugin, not a specific distro of Kubernetes.

Reply:
Too much better address it in the FAQ. Indeed we can give a better explanation. 👍